src/caches/abstractbeliefpropagationcache.jl

```diff
@@ -283,3 +283,69 @@ function update(
 )
   return update(Algorithm(alg), bpc; kwargs...)
 end
+
+function rescale_message(bp_cache::AbstractBeliefPropagationCache, partitionpair)
```

Comment:
Is `partitionpair` a partition edge? If so we should call it `partitionedge`.

Also, do these `rescale` functions always normalize? Maybe we should use the terminology `normalize` instead of `rescale`.

Reply:
Finally, it is a bit funny to me for this function to be singular, i.e. `rescale_message` instead of `rescale_messages`, if it is normalizing the messages in both direction along the partition edge.

The fact that you aren't specifying the type of the input makes me think maybe the function name should be more specific, i.e. `rescale_partition_edge_messages` or something like that.

Reply:
Yeah I believe that was the thinking we came up with from a while back: `normalize` is for an open tensor network and `rescale` for a closed one